src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -177,6 +183,14 @@ public AssemblyResultSet runLocalAssembly(final AssemblyRegion assemblyRegion,
         // print the graphs if the appropriate debug option has been turned on
         if ( graphOutputPath != null ) { printGraphs(nonRefSeqGraphs); }
         if ( graphHaplotypeHistogramPath != null ) { haplotypeHistogram.add((double)resultSet.getHaplotypeCount()); }
+        if ( debugActiveRegionOutputStream != null ) {
+            debugActiveRegionOutputStream.getNewLineBuilder()
+                .setRow(new String[]{
+                        assemblyRegion.getSpan().toString(),
```

Comment:
What would you think about a boolean option to use `getExtendedSpan`, which would give you the intervals (possibly overlapping) over which the GATK assembled?  I could imagine that being useful, too.

Reply:
I see two options here, both are potentially useful. I ask your opinion between the two. FIrst, I could add a second argument flag that dictates what level of padding gets included (which could possibly be expanded in the future). Second (and probably my preference) would be to add these as another column in the output of the file (say: unpadded_region) so the information would always be accessable.

Reply:
Let's go with the second option.

Reply:
it is now an extra column in every case. This will allow people to use either line they please if they use their bash magic to extract it, which is ultimately the goal here.